### ISSUE TITLE
Please explain the training steps using own dataset

### ISSUE THREAD

**Issue by DebojitBiswas** [chuanqi305/MobileNet-SSD#34]
Hi All,
I am new to SSD. Could someone please explain the step by step process to train the SSD model using my own dataset in caffe.
Thanks,
Debojit

**Comment by rangerToby** [comment]
hey, DebojitBiswas. I meet the same question, have you figure it out?

**Comment by DebojitBiswas** [comment]
Hi,
I have solved the problem. That was long time ago. I dont remember exactly. But I will try to help you. Please let me know your steps you performed and which step you are getting error.

**Comment by hammadullah125** [comment]
@DebojitBiswas I have to use SSD framework to train the network on my own image dataset. For that I looked in the internet how things work and how to train a SSD network. I found out that I need some files like "solver.prototxt" with the net defined inside solver "train.prototxt" and "test.prototxt" and a "labelmap.prototxt". There is also a "train_lmdb" and "test_lmdb" files needed. For generating lmdb files I did this 
$ GLOG_logtostderr = 1 / Users / screamer / caffe / build / tools / convert_imageset --resize_height = 256 --resize_width = 256 --shuffle / /Users/screamer/Desktop/Test/train.txt / Users / screamer / Desktop / Test / train_lmdb

where the "train.txt" file contains the path to the images and class number (format like this)
/export/users/hull/Downloads/cars_test/00001.jpg 0
/export/users/hull/Downloads/cars_test/00002.jpg 0
/export/users/hull/Downloads/cars_test/00003.jpg 0

In my case, I have just one class.

After doing all these necessary steps, I put the network on training
/Path/To/caffe train --solver /Path/To/solver.prototxt

But here I encounter a problem which is "Error parsing text-format caffe.NetParameter: 16:18: Message type "caffe.TransformationParameter" has no field named "resize_param".

I want to know that if I'm doing all this right? Where I'm making a mess? I would really be thankful if you help me out in explaining this.

**Comment by hammadullah125** [comment]
According to a github friend, there is no such functionality in caffe framework. To make it work, you would need to first get original caffe, and second merge it with https://github.com/ducha-aiki/caffe/tree/more-transforms
This process is non-trivial, so I don`t recommend you to do it. Instead, better use newer frameworks like pytorch or keras.
